tests/conftest.py

```diff
@@ -334,6 +337,21 @@ def client(testconfig, iaas, imageurl, request) -> Iterator[RemoteClient]:
         raise ValueError(f"invalid {iaas=}")
 
 
+def pytest_collection_modifyitems(config, items):
```

Comment:
Add a small description, why this `pytest_collection_modifyitems` hook is needed.

Reply:
Added a function documentation